examples/mnist-keras/README.md

```diff
@@ -52,4 +53,25 @@ bin/build.sh
 Finally, you can start the experiment from the "control" tab.
 
 ## Clean up
-You can clean up by running `docker-compose down`.
\ No newline at end of file
+You can clean up by running `docker-compose down`.
+
+## Connecting to a distributed deployment
+To start and remotely connect a client with the required dependencies for this example, start by downloading the `client.yaml` file. You can either navigate the reducer UI or run the following command.
+
+```bash
+curl -k https://<reducer-fqdn>:<reducer-port>/config/download > client.yaml
+```
+> **Note** make sure to replace `<reducer-fqdn>` and `<reducer-port>` with appropriate values.
+
+Now you are ready to start the client via Docker by running the following command.
+
+```bash
+docker run -d \
+  -v $PWD/client.yaml:/app/client.yaml \
+  -v $PWD/data:/var/data \
+  -e ENTRYPOINT_OPTS=--data_path=/var/data/mnist.npz \
+  --add-host=reducer:<reducer-ip> \
+  --add-host=combiner:<combiner-ip> \
+  ghcr.io/scaleoutsystems/fedn/fedn:develop-mnist-keras run client -in client.yaml
+```
+> **Note** make sure to replace `<reducer-ip>` and `<combiner-ip>` with appropriate values.
```

Comment:
Perhaps add here that --add-host is required if reducer and combiner names are not external dns allowing ingress.

Reply:
We do not need to map the reducer since the fqdn is set in the config file.